frontend/src/patientApp/timeOfTest/PatientProfileContainer.test.tsx

```diff
@@ -18,8 +18,8 @@ describe("PatientProfileContainer", () => {
   it("snapshot", () => {
     const store = mockStore({
       patient: {
-        residentCongregateSetting: "Yes",
-        employedInHealthcare: "Yes",
```

Comment:
Any considerations for existing data when this change lands?

Reply:
what existing data? my understanding is this will only effect the frontend which wont be picked up until the user refreshes the page and in that case they wont have any data